internal/pkg/docker/dockerengine/dockerengine.go

```diff
@@ -125,6 +127,14 @@ func (c CmdClient) Build(in *BuildArguments) error {
 		args = append(args, "--build-arg", fmt.Sprintf("%s=%s", k, in.Args[k]))
 	}
 
+	if in.ContainerName != "" {
```

Comment:
Is there a reason why you prefer setting the container label here instead if in `cli/deploy/workload.go`?

Reply:
The reason for setting the `container label` in `dockerengine.go` because 

I declared labels as `map[string]string`, but in [`cli/deploy/workload.go`](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/cli/deploy/workload.go#L366) we have a `for loop` to populate the `dockerengine.BuildArguments` 

if I do `labels["container"]= containerName` then the labels["container"] will always have the name of last Container. 
Is there any alternative that I can use?

Reply:
Maybe move the declaration of `labels` inside of the for loop?